components/kyma-environment-broker/internal/broker/instance_create.go

```diff
@@ -179,7 +179,7 @@ func (b *ProvisionEndpoint) Provision(ctx context.Context, instanceID string, de
 		ServiceID:       provisioningParameters.ServiceID,
 		ServiceName:     KymaServiceName,
 		ServicePlanID:   provisioningParameters.PlanID,
-		ServicePlanName: Plans(b.plansConfig, provisioningParameters.PlatformProvider, false)[provisioningParameters.PlanID].Name,
+		ServicePlanName: Plans(b.plansConfig, provisioningParameters.PlatformProvider, false, false)[provisioningParameters.PlanID].Name,
```

Comment:
in the plans.go there are PlanNamesMapping and PlanNamesMapping, it is better to use it here

Reply:
Done